internal/bminventory/inventory.go

```diff
@@ -2625,26 +2625,51 @@ func (b *bareMetalInventory) setOperatorsUsage(updateOLMOperators []*models.Moni
 }
 
 func (b *bareMetalInventory) updateClusterNetworkVMUsage(cluster *common.Cluster, updateParams *models.V2ClusterUpdateParams, usages map[string]models.Usage, log logrus.FieldLogger) {
-	vmHosts := make([]string, 0)
+	platform := cluster.Platform
+	usageEnable := true
```

Comment:
what is the meaning of this parameter? please add a comment

Reply:
what parameter? The platform could be from the persisted cluster or just changed by the UI. all the code has this fallback.

usageEnable: means enable(create) or disable(delete) this usage.